core/vm/instructions.go

```diff
@@ -635,9 +634,6 @@ func opCreate(pc *uint64, interpreter *EVMInterpreter, callContext *callCtx) ([]
 	}
 	callContext.contract.Gas += returnGas
 
-	if suberr == ErrExecutionReverted {
```

Comment:
So, here the assumption is made the when `interpreter.evm.Create` returns `ErrExecutionReverted` in the last return value, the `res` is always `nil`. Need to check that.

Reply:
Or is it that the `res` of the reverted `Create` is not useful?

Reply:
I think it wasn't being used in the calling code, so I took it that it wasn't useful.